lib/author.rb

```diff
@@ -0,0 +1,23 @@
+class Author
+  attr_reader :first_name,
+              :last_name,
+              :name,
+              :books
+
+  def initialize(params = {})
+    @first_name = params.fetch(:first_name)
+    @last_name  = params.fetch(:last_name)
+    @name       = full_name(:first_name, :last_name)
```

Comment:
I would create a method called name, doing this you are storing redundant information